lib/Doctrine/DBAL/Platforms/SQLServerPlatform.php

```diff
@@ -192,7 +192,7 @@ public function getDropIndexSQL($index, $table = null)
 
         return "IF EXISTS (SELECT * FROM sysobjects WHERE name = '$index')
```

Comment:
@stof Ditto - I'll see if I can get some of the failure scenarios fixed or at least tested for.